### PR TITLE
Enable display mode setting

### PR DESCRIPTION
**Pull request by Nickbert7**
image

Hosted preview with some other changes: https://nickbert7.github.io/jellyfin-web/src/#!/selectserver.html

### CODE REVIEW

**Comment by Nickbert7** [comment]
In TV it shows an additional home screen setting:
image

But so far it seems not to have any effect...

**Comment by EraYaN** [comment]
The horizontal home screen seems to be completely missing for some reason.

File: src/components/focusManager.js [hunk 1]
```diff
@@ -40,7 +40,7 @@ define(['dom'], function (dom) {
 
         try {
             element.focus({
-                preventScroll: true
+                preventScroll: false
```
Comment (EraYaN) [inline src/components/focusManager.js]: Maybe this fixes some the the webOS problems? @oddstr13

Comment (dkanada) [reply src/components/focusManager.js]: According to chat logs this didn't fix anything for WebOS.

**Pull request merged**